2019-05-15-all-hands-meeting.md

```diff
@@ -0,0 +1,158 @@
+# 2019-05-15 Toronto Workers Co-op: All-Hands Meeting
+
+<sup>[from template][template]</sup>
+
+[:date: Calendar][cal] | [:cat: GitHub][gh] | [:open_file_folder: GDrive][gdrive]
+
+Time: 17:00 - 18:00 EDT  
+Location: https://meet.jit.si/offline  
+Attending: 
+- yurko
+- elon
+- garry
+- dc (cultivator)
+- ben
+
+Notetaker:  
+- [ ] benedict  
+- [~] dc  
+- [~] elon  
+- [~] garry  
```

Comment:
Ok, I've gotta ask -- what does this mean? Is it, people who've done it already?

Reply:
yeah, because we immediately got out of sync it was a way to see who had taken notes